documentation/guides/advanced-guides/multiple-browser-windows.html

```diff
@@ -0,0 +1,1221 @@
+
+<a href="/testcafe/documentation/guides/continuous-integration/azure-devops.html"  >Azure DevOps</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/bitbucket-pipelines.html"  >Bitbucket Pipelines</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/circleci.html"  >CircleCI</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/github-actions.html"  >GitHub Actions</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/github-actions-and-browserstack.html"  >GitHub Actions + BrowserStack</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/gitlab.html"  >GitLab</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/jenkins.html"  >Jenkins</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/teamcity.html"  >TeamCity</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/travis.html"  >Travis</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/continuous-integration/travis-and-sauce-labs.html"  >Travis + Sauce Labs</a>
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+    <li>
+		    
+		
+
+<span>Extend TestCafe</span>
+
+		
+			<ul>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/extend-testcafe/install-plugins.html"  >Install Plugins</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/extend-testcafe/reporter-plugin.html"  >Reporter Plugin</a>
+
+		
+    </li>
+
+    <li>
+		    
+		
+
+<a href="/testcafe/documentation/guides/extend-testcafe/browser-provider-plugin.html"  >Browser Provider Plugin</a>
+
+		
+    </li>
+
+</ul>
+		
+    </li>
+
+</ul>
+				</li>
+			</ul>
+		</div>
+	</nav>
+	
+
+	
+	
+
+	
+	
+	
+
+
+<main>
+	<article class="doc">
+		<div class="post-content">
+
+			<h1>Multiple Browser Windows</h1>
+<p>The TestCafe API includes methods that open, close, and switch between browser windows. You can test websites with pop-up windows and OAuth login forms, debug complex multi-window applications, or run multiple instances of the same web app side-by-side.</p>
+
+<p>⚠ This is a <strong>beta</strong> feature. Browser support is limited to local instances of Chrome and Firefox. You cannot capture videos or screenshots of child windows. The available functionality is subject to further revisions. Please do not use this feature in production environments.</p>
+
+<p>️🛠️ Use the <code>--disable-multiple-windows</code> CLI flag to disable support for multiple browser windows if you encounter compatibility issues with <a href="http://testcafe.devexpress.com/documentation">TestCafe v2015.1</a> tests.</p>
```

Comment:
> if you encounter compatibility issues with TestCafe v2015.1 tests

Not only for them. There are problems with tests written with the new open-source version. Customers can encounter some problems run tests on cloud browsers.